frontend/src/env.ts

```diff
@@ -39,5 +39,29 @@ export const USER_INFO_URL: string = `${BASE_URI}/auth/userinfo`
 export const LOGOUT_URL: string = `${BASE_URI}/auth/logout`
 export const USER_DELETION_URL: string = `${BASE_URI}/auth/deleteuser`
 
+export const WALLET_BASE_URL: string = `${BASE_URI}/wallet`
+export const REMOVE_CARD_URL: string = `${WALLET_BASE_URL}/removecard`
+export const WALLET_INFO_URL: string = `${WALLET_BASE_URL}/walletinfo`
+export const TRANSACTIONS_URL: string = `${WALLET_BASE_URL}/transactions`
+export const STRIPE_DATA_URL: string = `${WALLET_BASE_URL}/stripedata`
+export const TRANSACTION_INFO_URL= (transaction: string) => {
```

Comment:
Formatting seems off

Reply:
Interesting, setting the default formatter in VS Code for `[javascript]` handles .tsx files, but I forgot to set the same `[typescript]`